src/components/cms/BotBuilder/Preview/Preview.js

```diff
@@ -461,7 +486,7 @@ class Preview extends Component {
                                     rows="1"
                                     value={message}
                                     onKeyPress={event => {
-                                      if (event.which === 13) {
+                                      if (event.which === ENTER_KEY_CODE) {
```

Comment:
A util can be made for `onChatComposerKeyDown` and be reused. It will reduce redundancy more.